reframe/frontend/executors/__init__.py

```diff
@@ -461,7 +467,11 @@ def on_task_skip(self, task):
 
     @abc.abstractmethod
     def on_task_failure(self, task):
-        '''Called when a regression test has failed.'''
+        '''Called when a RegressionTask has failed.'''
+
+    @abc.abstractmethod
+    def on_task_abort(self, task):
+        '''Called when a RegressionTask has failed.'''
```

Comment:
```suggestion
        '''Called when a RegressionTask has aborted.'''
```
Maybe you meant to write aborted here?